packages/components/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "px-to-rem.px-per-rem": 16
```

Comment:
is this an extension that converts from pixels to rem?

Reply:
yes, it is! i've added by mistake the .vscode folder to the repo